instrumentation/p6spy/src/test/java/brave/p6spy/TracingJdbcEventListenerTest.java

```diff
@@ -66,4 +78,22 @@
 
     verifyNoMoreInteractions(span);
   }
+
+
+  @Test public void nullSqlWontNPE() throws SQLException {
+    ConcurrentLinkedDeque<zipkin.Span> spans = new ConcurrentLinkedDeque<>();
+    Tracing tracing = ITTracingP6Factory.tracingBuilder(Sampler.ALWAYS_SAMPLE, spans).build();
+
+    when(statementInformation.getSql()).thenReturn(null);
+    when(statementInformation.getConnectionInformation()).thenReturn(ci);
+    when(ci.getConnection()).thenReturn(connection);
+    when(connection.getMetaData()).thenReturn(metaData);
+    when(metaData.getURL()).thenReturn(url);
+
+    TracingJdbcEventListener listener = new TracingJdbcEventListener("", false);
+    listener.onBeforeAnyExecute(statementInformation);
+    listener.onAfterAnyExecute(statementInformation,1,null);
+
+    assertThat(spans.getFirst().name, is(equalTo(TracingJdbcEventListener.QUERY_PLACEHOLDER.trim())));
```

Comment:
nit: prefer assertj since we use it everywhere. assertThat(...).isEqualTo(

Also, best to use the string constant vs having people look it up :) a touch of anti DRY in tests can make it easier to understand.